Add a --help option.

craig-spannring opened the pull request:
Small change to the command line argument parser.  If given a bogus
option the program will print an error message and exit with exit code 
of 1.  If given "--help" or "-h" it will print a short description of the 
allowed command line arguments and exit with exit code 0. 

Add VS Code launch configurations to test different command line arguments.

craig-spannring committed:
Add a --help option.

Add VS Code launch configurations to test different command line
arguments.

frank-weinberg commented on .vscode/launch.json:
```diff
@@ -11,6 +11,72 @@
+    },
+    {
+      "type": "java",
+      "name": "scoreboard --host=bogus",
```
that's in there twice

craig-spannring replied:
One of those tests the invalid option "--bogus".  The other tests with a (hopefully) invalid hostname.

frank-weinberg commented on src/com/carolinarollergirls/scoreboard/Main.java:
```diff
@@ -138,7 +138,22 @@ private void parseArgv(String[] argv) {
-            }
+            } else if (arg.equals("--help") || arg.equals("-h")) {
+                System.out.println("Options:");
+                System.out.println("  --gui, -g                  (default) Create GUI window to show program messages.");
```
The default is --nogui. (The windows starter has the --gui flag embedded.)

craig-spannring committed:
Fix usage message.

The default is --nogui, not --gui.

craig-spannring committed:
Use hostname bogus.example.com.

The host bogus.example.com is almost guaranteed to not exist,
whereas bogus might be a valid host in your local DNS server.

craig-spannring commented:
I've made some changes that should address your concerns.
